app/controllers/general_shopping_list_controller.rb

```diff
@@ -0,0 +1,19 @@
+class GeneralShoppingListController < ActionController::Base
+  include GeneralShoppingListHelper
+
+  layout 'application'
+
+  def index
+    @ingredients = getting_shopping
```

Comment:
Good job using verbs to define method names :)

Just remember that infinitive verbs are most recommended.

Reply:
Thanks for your review, I used infinitive verbs for the method, but rubocop doesn't let start methods with `get_`